internal/pkg/describe/webapp.go

```diff
@@ -296,6 +336,28 @@ func (w *WebApp) HumanString() string {
 	for _, route := range w.Routes {
 		fmt.Fprintf(writer, "  %s\t%s\t%s\n", route.Environment, route.URL, route.Path)
 	}
+	fmt.Fprintf(writer, color.Bold.Sprint("\nVariables\n\n"))
+	writer.Flush()
+	fmt.Fprintf(writer, "  %s\t%s\t%s\n", "Name", "Environment", "Value")
+	var prevName string
+	var prevValue string
+	for _, variable := range w.Variables {
+		if variable.Name != prevName {
+			if variable.Value != prevValue {
+				fmt.Fprintf(writer, "  %s\t%s\t%s\n", variable.Name, variable.Environment, variable.Value)
+			} else {
+				fmt.Fprintf(writer, "  %s\t%s\t-\n", variable.Name, variable.Environment)
```

Comment:
Can we add comments to these blocks to explain what's happening:
```go
// Instead of re-writing the same variable value, we replace it with "-" to reduce text.
```